src/main/groovy/org/labkey/gradle/plugin/TeamCity.groovy

```diff
@@ -350,6 +350,10 @@ class TeamCity extends Tomcat
                     {
                         arg(line: "-q firefox")
                     }
+            project.ant.exec(executable: "killall")
+                    {
+                        arg(line: "-q firefox-bin")
+                    }
```

Comment:
Is there a similar kind of discrepancy in executable naming for Windows?

Reply:
There doesn't seem to be. Checked a running agent and only saw `firefox`, no `firefox-bin` or similar